core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -56,9 +53,7 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
   <div>
     <background-banner></background-banner>
     <conversation-skin></conversation-skin>
-    {% if not iframed %}
-      <attribution-guide></attribution-guide>
-    {% endif %}
+    <attribution-guide ng-if="!iframed"></attribution-guide>
```

Comment:
Just to check, where is $scope.iframed defined?

Reply:
Ah ok. That seems a bit indirect and relies on controllers knowing about their parent scopes.

Could we extract the "isIframed" functionality into a service since we're introducing a dependency on it here, and then have the exploration player directive/controller get the value from that service instead? I'm uncomfortable with bringing this PR in without that, since you kind of have to guess at where $scope.iframed is located. In general the variables used in a template should be defined within the closest directive or controller.

Reply:
The problem is that both place where I use `ng-if` is directly are directly under Base.js (there is no controler under Base that modifies ` $scope.iframed`). And both parts can't be part of one controler, because they're in different Jinja blocks.

So, should I create two new directives for each place? Also, `$scope.iframed` is retrieved from GLOBALS we will have to do some modifications later when we remove it from GLOBALS.